pygameMenu/menu.py

```diff
@@ -588,6 +580,17 @@ def add_text_input(self,
 
         return widget
 
+    def set_initial_values(self, align, font_size):

```

Comment:
You means `get_initial_values` ? (this function set nothing)

Reply:
Sorry for my short comment.
In fact this function is quite ambiguous, it update the widget global position and it process the alignment and font size.

Maybe should be split un 2 private methods

Reply:
Good idea. I'll change it and resubmit.